openslides/users/static/templates/users/user-change-password.html

```diff
@@ -0,0 +1,50 @@
+<div class="header">
+  <div class="title">
+    <div class="submenu">
+      <a ui-sref="users.user.list" class="btn btn-sm btn-default">
+        <i class="fa fa-angle-double-left fa-lg"></i>
+        <translate>Back to overview</translate>
+      </a>
+    </div>
+    <h1 translate>Change password for {{ user.get_short_name() }}</h1>
+  </div>
+</div>
+
+<div class="details">
+  <form name="userForm">
+    <h3 class="text-danger" style="margin-top: 0;">
+      <i class="fa fa-exclamation-triangle"></i>
+      <translate>You override a personally set password from</translate> {{ user.username }}!
```

Comment:
`<translate>You override the personally set password of</translate> {{ user.get_short_name() }}!`